Source/Core/Mixin.lua

```diff
@@ -76,6 +76,22 @@ local function InitializeAuctionatorButtonFrame()
   AuctionatorButtonFrame:Show()
 end
 
+local function InitializeSellingFrame()
+  local frame
+  if Auctionator.State.SellingFrameRef == nil then
+    frame = CreateFrame(
+      "FRAME",
+      "AuctionatorSellingFrame",
+      AuctionHouseFrame,
+      "AuctionatorSellingFrameTemplate"
+    )
+
+    Auctionator.State.SellingFrameRef = frame
+  else
+    frame = Auctionator.State.SellingFrameRef
+  end
```

Comment:
this can be deleted

Reply:
Why?

Reply:
We don't use the value of frame outside of this function. I meant to only select the last 3 lines, sorry